test/util_test.py

```diff
@@ -25,3 +28,35 @@ def test_accepted_unit(self):
 
         with pytest.raises(TypeError):
             parse_unit("1:ul", ["second", "kg"])
+
+
+class TestUtil(object):
+    def test_compatible_reservoir_container(self):
+        # asserts that no exception is raised. If it raises an exception, we catch it, display it, and assert False.
+        try:
+            container_type = _CONTAINER_TYPES["res-sw384-lp"]
+            shape = LiquidHandle.builders.shape(1, 1, "SBS384")
+            _check_container_type_with_shape(container_type, shape)
+        except Exception as exc:  # pylint: disable=W0703
+            assert False, f"{exc}"
+        # asserts that an exception is raised of class ValueError
+        with pytest.raises(ValueError):
+            container_type = _CONTAINER_TYPES["res-sw96-hp"]
+            shape = LiquidHandle.builders.shape(1, 1, "SBS384")
+            _check_container_type_with_shape(container_type, shape)
+
+    def test_stamp_single_well_to_384_well_reservoir(self):
+        # asserts that no exception is raised. If it raises an exception, we catch it, display it, and assert False.
+        try:
+            p = Protocol()
+            source = p.ref("source", cont_type="res-sw384-lp", discard=True)
+            destination = p.ref("destination", cont_type="384-flat", discard=True)
+            p.transfer(
+                source=source.well(0),
+                destination=destination.well(0),
+                volume="10:microliter",
+                rows=16,
+                columns=24,
+            )
+        except Exception as exc:  # pylint: disable=W0703
```

Comment:
I think I may have read this incorrectly above...

What is the _expected behavior_ here? Both methods in this test class `TestUtil` should **not** cause errors, right? I think wrapping these in try/except isn't the easiest approach.

Might I suggest something like:
1. One test that shows that we can transfer from 384-well plate -> 1-well reservoir
2. Another test showing that we still can't transfer from 384-well plate -> 96-well plate.... just to demonstrate that we're still hitting that expected Error because we're lacking `sbs384_ability` or whatever